modules/openapi-generator/src/main/resources/kotlin-client/interface_opt_var.mustache

```diff
@@ -2,9 +2,7 @@
     /* {{{description}}} */
 {{/description}}
     {{^multiplatform}}
-    {{#moshi}}
-    @Json(name = "{{{vendorExtensions.x-base-name-literal}}}")
-    {{/moshi}}
+    {{#moshi}}{{^discriminator}}@Json(name = "{{{vendorExtensions.x-base-name-literal}}}"){{/discriminator}}{{/moshi}}
```

Comment:
```
     {{#moshi}}
     {{^discriminator}}@Json(name = "{{{vendorExtensions.x-base-name-literal}}}"){{/discriminator}}
     {{/moshi}}
```

to avoid blank line in the output.

Reply:
Changed. Thanks for the hint :+1: